Shared/Flows/Onboarding/PhotoViewController.swift

```diff
@@ -102,7 +102,9 @@ class PhotoViewController: ViewController, Sizeable, Completable {
         self.confirmButton.set(style: .normal(color: .purple, text: "Continue"))
         self.confirmButton.didSelect { [unowned self] in
```

Comment:
Selectable would also be a great place to have async await. No more closures where we forget to call unowned self.